server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java

```diff
@@ -232,13 +232,14 @@ public static boolean reaudit(final CountyDashboard cdb,
                                 final String comment) {
 
     LOGGER.info("[reaudit] cvr: " + cvr.toString());
-    final CVRAuditInfo cai =
-      Persistence.getByID(cvr.id(), CVRAuditInfo.class);
-    final CastVoteRecord oldAcvr = cai.acvr();
-    if (null == oldAcvr) {
+
```

Comment:
We could perhaps use DD or Democracy Developers rather than individual initials?